src/components/Navbar.js

```diff
@@ -1,15 +1,42 @@
-import React from "react"
+import React, { Component } from "react"
 import { Link } from "react-router-dom"
+import "./Navbar.css"
 
-const Navbar = () => {
-    return (
-        <div className="Navbar">
-            <li><Link to="/excursions">Excursion</Link></li>
-            <li><Link to="/transport">Transportation</Link></li>
-            <li><Link to="/contact">Contact</Link></li>
-            <li><Link to="/signup">Sign up</Link></li>
-        </div>
-    )
-}
+class Navbar extends Component {
+    constructor() {
+        super()
+        this.state = {
+            user: "organizer"    // For the displayed menu items depend on this 'user' state.
+        }
+    }
+
+    render() {
 
+        if (this.state.user == "none") {
+            return (<div className="Navbar">
+                <Link to="/">Start</Link>
+            </div>)
+        } else if (this.state.user == "participant") {
+            return (
+                <div className="Navbar">
+                    <Link to="/excursions">Excursion</Link>
+                    <Link to="/transport">Transportation</Link>
+                    <Link to="/contact">Contact</Link>
+                    <Link to="/signup">Sign up</Link>
+                    <Link to="/">Start</Link>
+                </div>
+            )
+        } else if (this.state.user == "organizer") {
+            return (
+                <div className="Navbar">
+                    <Link to="/excursions">Excursion</Link>
+                    <Link to="/participantList">Participant List</Link>
+                    <Link to="/dutyList">Duty List</Link>
+                    <Link to="/shoppingList">Shopping List</Link>
+                    <Link to="/">Start</Link>
+                </div>
```

Comment:
Don'tRepeatYourself :)
Limit your differences between two boolean branches (if-else) to only what actually differs. Here you have the containing "NavBar" div, Excursion, and Start links on both branches.

On a different notion. Do your organizers not need to also sign up for excursions? Currently, if the user is organizer they have no access to the sign up page :)